src/main/resources/schema.sql

```diff
@@ -16,6 +16,7 @@ CREATE TYPE error_code AS ENUM(
   'GRPC_UNAVAILABLE',
   'GRPC_DATA_LOSS',
   'GRPC_UNAUTHENTICATED',
+  'PROJECT_NOT_FOUND',
```

Comment:
Asking just in case. Is it safe to reorder the enum constants? e.g. `AUTOSCALER_INTERNAL` moved on step now.